src/components/document/document-workspace.tsx

```diff
@@ -17,7 +17,6 @@ import "./document-workspace.sass";
 type WorkspaceSide = "primary" | "comparison";
 
 interface IProps extends IBaseProps {
-  isGhostUser: boolean;
```

Comment:
Is this interface needed at all now?

Reply:
Not really, but by convention all components have their own `IProps` interface. In this case it serves as a placeholder in case it ever acquires new props so I didn't feel compelled to remove it.